### PR TITLE
[ACQ-552] Style filter dropdowns on /curriculum (pt.2)

### PR DESCRIPTION
**Pull request by levadadenys**
### [ACQ-552] Style filter dropdowns on /curriculum (pt.2)
  * update curiculum catalog filters styles
  
  Before:
Знімок екрана 2023-06-07 о 22 48 50

  After:
Знімок екрана 2023-06-07 о 22 46 00

## Links
- jira ticket: [ACQ-554](https://codedotorg.atlassian.net/browse/ACQ-554)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/checkbox-dropdown.module.scss [hunk 1]
```diff
@@ -1,25 +1,54 @@
 @import "color.scss";
 
+/* stylelint-disable */
+// checkbox active background color
+$light_primary_500: #0093A4;
```
Comment (TurnerRiley) [inline apps/src/templates/checkbox-dropdown.module.scss]: Could you add this color to `color.scss` please, then you wouldn't need the disable lint line above it?

File: apps/src/templates/CheckboxDropdown.jsx [hunk 2]
```diff
@@ -55,19 +56,13 @@ const CheckboxDropdown = ({
...
-                  {allOptions[optionKey]}
-                </Typography>
-              </label>
+              <Checkbox
+                checked={checkedOptions.includes(optionKey)}
+                onChange={onChange}
+                name={optionKey}
+                value={optionKey}
+                label={allOptions[optionKey]}
+              />
```
Comment (TurnerRiley) [inline apps/src/templates/CheckboxDropdown.jsx:65]: Thank you for simplifying this! This looks so much better and is much more readable

**Review by TurnerRiley — APPROVED** [review]

File: apps/src/templates/checkbox-dropdown.module.scss (hunk 1 above)
Comment (levadadenys) [reply apps/src/templates/checkbox-dropdown.module.scss:5]: That's intended, but thanks for noticing! The color is part of our new colors set that's coming soon. I've also used it in the same way in checkbox component for now. Once new colors set will be ready and on staging, these color declarations will be deleted

Comment (levadadenys) [reply apps/src/templates/checkbox-dropdown.module.scss]: That was intended, but indeed you're rigth, thank you!
This color is part of our new colors set that's coming soon. The initial idea was to save that colors in scss modules until full set of colors i released, but I agree that it'll be easier to store them in colors.scss anyways

**Pull request merged**